### PR TITLE
Add temporary UI for method creation

### PR DESCRIPTION
**Pull request by zerox1212**
Fixed UI which allows adding a single input argument

Use this code to test my other PRs for supporting Methods in UA Modeler.

GUI needs to be finalized before merge to master!

### CODE REVIEW

File: uawidgets/new_node_dialogs.py [hunk 1]
```diff
@@ -137,16 +137,48 @@ def get_args(self):
...
 
-
 class NewUaMethodDialog(NewNodeBaseDialog):
     def __init__(self, parent, title, server):
         NewNodeBaseDialog.__init__(self, parent, title, server)
         # FIXME: design UI!
 
+        # FIXME below widgets need to be encapsulated into a class which can be created dynamically,
+        # FIXME perhaps with a plus button or something; get_args then needs to loop through the created
+        # FIXME widgets to collect all argument info for ua.Argument creation
```
Comment (oroulet) [inline uawidgets/new_node_dialogs.py]: correct vision :+1:

**Comment by zerox1212** [comment]
I'm going to try and do a basic UI. If you have already started let me know, otherwise I will continue.

**Comment by zerox1212** [comment]
How can I make a "+" button which calls add_row() function?
Also, how can I make my separator line full width?
image

**Comment by oroulet** [comment]
You should use a listview or tree view and override the default widget. Then add a button outside, that add new item when clicked. Google for examples

**Comment by oroulet** [comment]
Looks good

**Comment by zerox1212** [comment]
I'm running out of time to work on this. For now I will just make 3x in and out arguments fixed. If name is blank it will be skipped. This meets my current needs for now if that's OK with you.

If I get time I will learn how to use listview and treeview. Sorry, this is my first time doing PyQT.

**Comment by zerox1212** [comment]
You can merge this now if you want. Temporary solution looks like this:
image

If the Name field is blank the argument is ignored.

**Comment by oroulet** [comment]
Looks good! I do not have time to work on this for the time being, so feel
free to learn listview api :-)
Oh and put the ok cancel buttons on bottom if you can

**Comment by zerox1212** [comment]
It isn't worth moving the buttons to the bottom in my opinion. Buttons are added to the layout in `NewNodeBaseDialog`. I don't know an easy way to override that and put it in a new vertical layout at the bottom... I think the buttons would have to be moved to every subclass if you really want this.

**Pull request merged**